### DISCUSSION TITLE
Проблемы с работой из docker-контейнера

### DISCUSSION THREAD

**Discussion by skr7andy** [hufrea/byedpi#110] [Q&A]
Установлен на роутер микротик (ARM64) [docker-контейнер](https://hub.docker.com/r/tazihad/byedpi) с параметрами запуска `--disorder 1 --tlsrec 1+s --debug 1`. С браузера подключаюсь к прокси, ютуб открывается медленно, видео еле грузится в 144p, Connection speed ~ 400Kbps.
Точной такой же контейнер, с такими же параметрами запуска установлен на Synology NAS (x86) - всё летает, 4к почти без задержки перематывается. 
Подскажите, куда смотреть, в чём может быть пробема?
При этом, если не использовать никакой прокси, видео на ютубе не загружаются совсем - чёрный экран.

Скриншот логов контейнера с микротика
![Screenshot_20240830_124434](https://github.com/user-attachments/assets/81fc931a-7baa-4c91-911a-1738b7453df3)

**Comment by wiktorbgu** [comment]
Можете поробовать настроить по инструкции https://habr.com/ru/articles/838452/ (доступна через vpn или в вебархиве)
Есть минимизированный контейнер byedpi https://hub.docker.com/r/wiktorbgu/byedpi-mikrotik
Или попробуйте контейнер https://hub.docker.com/r/wiktorbgu/byedpi-hev-socks5-tunnel указать в качестве туннеля
Во всех вариантах нет проблем, ютуб летает, роутер hAP ax3. 
Возможно нужно сменить команду byedpi, для автоподбора есть скрипт https://ntc.party/t/goodcheck-блокчек-скрипт-для-goodbyedpi-zapret-byedpi/
